Content/Guides/manpages/pdbtool.1.html

```diff
@@ -295,9 +295,8 @@ <h6>Files</h6>
 		</div>
 		<div>
 			<h6>See also</h6>
-			<p><MadCap:xref href="https://www.syslog-ng.com/"><span style="color: #04aada;" class="mcFormatColor">The syslog-ng
-					Administrator Guide</span>
-				</MadCap:xref>
+			<p><a href="https://www.syslog-ng.com/"><span style="color: #04aada;" class="mcFormatColor">The syslog-ng
```

Comment:
Is the gap here lots of whitespaces, or does it only look like that in this merge request view?  (The same question for all occurrences.)

Reply:
fixed in every manpage